File: src/main/java/no/stunor/origo/batch/data/EventorRepository.java
package no.stunor.origo.batch.data;

import org.springframework.stereotype.Repository;

import com.google.cloud.spring.data.firestore.FirestoreReactiveRepository;

import no.stunor.origo.batch.model.Eventor;

@Repository
public interface EventorRepository extends FirestoreReactiveRepository<Eventor> {
}

File: src/main/java/no/stunor/origo/batch/data/EventorRepository.kt
package no.stunor.origo.batch.data

import com.google.cloud.spring.data.firestore.FirestoreReactiveRepository
import no.stunor.origo.batch.model.Eventor
import org.springframework.stereotype.Repository

@Repository
interface EventorRepository : FirestoreReactiveRepository<Eventor>